lib/lookup.json

```
@@ -24,6 +24,13 @@
     "prefix": "v",
     "flaky": true
   },
+  "@yarnpkg/cli": {
+    "maintainers": "arcanis",
```

```suggestion
    "maintainers": ["arcanis", "merceyz", "paul-soporan", "larixer"],
```
cc @paul-soporan @larixer